### DISCUSSION TITLE
blog/2023/07/03/using-google-location-history-with-r-roadtrip/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#86] [Blog comments]
# blog/2023/07/03/using-google-location-history-with-r-roadtrip/index

Learn how to use R to load and clean and play with all the location history data Google keeps about you and look at some neat plots and tables about our 5,000-mile summer road trip along the way

https://www.andrewheiss.com/blog/2023/07/03/using-google-location-history-with-r-roadtrip/index.html

**Comment by fdporter1850** [comment]
Thanks so much for this!  Very well done.  Now I need to find time to play with it:)

**Comment by rtol** [comment]
Wonderful -- until Google changed its data format.

Code crashes at "pluck locations" ...

**Comment by trashstories** [comment]
This has been so helpful in using my Google location data, but now that Google has removed location history from the takeout options. I figured out you can still get the data through a convoluted process, but it is now Timeline.json instead of Records.json, so is there any chance this post will be updated with a new process to deal with the new format?
